trident/data/line_lists/lines.txt

```diff
@@ -49,7 +50,7 @@ C I            1193.031000    6.390000e+07    4.090000e-02
 C I            1188.833000    1.950000e+07    1.240000e-02                
 C I            1157.910000    3.520000e+07    2.120000e-02                
 C II           1335.663000    4.760000e+07    1.270000e-02      C II* 1336
-C II           1334.532000    2.410000e+08    1.290000e-01                
+C II           1334.532000    2.400000e+08    1.280000e-01                
```

Comment:
@chummels, can you comment on these changes to the line data?

Reply:
I'm not sure why these changes are being made, especially given that this PR is dealing with H I, not C II.  Furthermore, looking at the Morton tables, this change looks to be incorrect.  I think this change should be removed.

Reply:
These changes are in line with the Morton data, but it just seems odd to include these minor changes in a PR that's about depositing HI as 21 cm lines.  But they can stay, as I don't think they'll break the answer tests.  I don't *think* so anyway.